kokoro/android/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright (c) 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Android Build Script.
+
+# Fail on any error.
+set -e
+# Display commands being run.
+set -x
+
+BUILD_ROOT=$PWD
+SRC=$PWD/github/SPIRV-Tools
+TARGET_ARCH="armeabi-v7a with NEON"
+export ANDROID_NDK=/opt/android-ndk-r15c
+
+# Get NINJA.
+wget -q https://github.com/ninja-build/ninja/releases/download/v1.7.2/ninja-linux.zip
```

Comment:
Kokoro does not have ninja installed by default?

The latest is v1.8.2, which is released on 2017-09-12: https://github.com/ninja-build/ninja/releases/tag/v1.8.2

Also for other places.

Reply:
Done